rxfirebase/src/main/java/com/ezhome/rxfirebase2/database/RxFirebaseDatabase.java

```diff
@@ -58,6 +64,15 @@ private RxFirebaseDatabase() {
   }
 
   /**
+   * This method will set specific Scheduler on what values will be Observed on
+   * @param observeOnScheduler {@link Scheduler} for observed on
+   */
+  public RxFirebaseDatabase setObserveOnScheduler(Scheduler observeOnScheduler) {
```

Comment:
I would name it as `observeOn(Scheduler)`. Everyone already knows that they need to setup the scheduler.

Update also the wiki page to make clear to the community that they have the flexibility to setup their own `observeOn` scheduler.